docs/docs-ref-autogen/excel/excel.chartaxis.yml

```diff
@@ -158,7 +158,7 @@ items:
           #### Examples
 
           ```typescript
-          await Excel.run(async (context) => {
+          await Excel.run(async context => {

```

Comment:
Alex is cleaning up these async context items. I'll merge after I get that update.

Reply:
Need to have another discussion as Script Lab is stripping out these non-whitespace characters. I'll go ahead and check this in in the meantime.